adafruit_tfmini.py

```diff
@@ -87,24 +93,24 @@ def distance(self):
         raise RuntimeError("Timed out looking for valid data")
 
     @property
-    def strength(self):
+    def strength(self) -> int:
         """The signal validity, higher value means better measurement"""
         _ = self.distance  # trigger distance measurement
         return self._strength
 
     @property
-    def mode(self):
+    def mode(self) -> int:
         """The measurement mode can be MODE_SHORT (2) or MODE_LONG (7)"""
         _ = self.distance  # trigger distance measurement
         return self._mode
 
     @mode.setter
-    def mode(self, newmode):
+    def mode(self, newmode: int) -> None:
         if not newmode in (MODE_LONG, MODE_SHORT):
             raise ValueError("Invalid mode")
         self._set_config(_CONFIGPARAM + bytes([0, 0, newmode, 0x11]))
 
-    def _set_config(self, command):
+    def _set_config(self, command: int) -> None:
```

Comment:
`command` can be `ReadableBuffer` from `circuitpython_typing` from `adafruit-circuitpython-typing`.